### DISCUSSION TITLE
Suchfeld bei "weitere Karten"

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#72] [Q&A]
Es geht um die beiden auf dem Screenshot umrandeten Möglichkeiten zwischen den Karten zu wechseln:

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/ee124cec-755c-49d8-ba44-ee03ee069876)

Ist es möglich bei der Option "weitere Karten" ganz oben ein Suchfeld anzubieten, gleich wie bei der Option "Portal: portal"?

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/ae5c37ff-aec6-4594-af48-3cd3eb4d0a22)

Da sich die beiden Funktionen nur dahingehend unterscheiden, dass beim Wechsel entweder der Kartenausschnitt erhalten bleibt oder eine  neue Karte geöffnet wird, würde es es sich anbieten bei "weitere Karten" ein Suchfeld einzubauen. In dem Fall hätte man dann die Möglichkeit nur eine der beiden Optionen zum Wechseln der Karten anzubieten. Spart ausserdem noch Platz auf Handy u Tablet.

**Comment by nicole-vlbg** [comment]
Reaktivierung der noch auf "unbeantwortet" stehenden Anfrage:
Gibt es zu dem nicht vorhandenen Suchfeld unter "weitere Karten" in WebGIS6 evtl. eine Lösung?
Vorschlag siehe oben

**Reply by gis-eni** [reply]
Hallo,

wird leider im WebGIS 6 nicht mehr kommen. Bei WebGIS 7 steht es noch auf der Agenda.

**Reply by ue-black** [reply]
noch eine kurze Nachfrage: Kann man den oben eingerahmten Text viell. auch ändern oder änderbar machen?
 z.B. "zurück zum Kartenportal" und "Karte wechseln"...

**Reply by gis-eni** [reply]
Wird (oder ist) mit WebGIS 7 möglich. Da der Viewer in WebGIS 7 mehrsprachig angeboten wird, kannst du die Texte theoretisch ändern. Ist allerdings noch nicht dokumentiert.
